internal/runtime/client/client.go

```diff
@@ -320,6 +330,7 @@ func (c *client) CallExtension(ctx context.Context, hook runtimecatalog.Hook, fo
 		return errors.Errorf("failed to call extension handler %q: namespaceSelector did not match object %s", name, util.ObjectKey(forObject))
 	}
 
+	log.Info(fmt.Sprintf("Calling extension %q of hook %q", name, hookName))
```

Comment:
Thoughts about adding `extension` and `hook` as k/v pairs to the log at the top of the function so that we dont have to repeat the line `extension %q of hook %q` in multiple logs? 
This might affect readability of the logs.

Reply:
Sounds good, we should also set hook at the beginning of `CallAllExtensions`. It's not an issue if we overwrite that then in CallExtensions.

Reply:
Proposal for k/v pairs:
* extensionHandler > name
* hook => hookName
WDYT?

(we can adjust later on if necessary once we've established logging conventions)